examples/angular-dapp/src/app/app.component.ts

```diff
@@ -1,10 +1,59 @@
 import { Component } from '@angular/core';
+import axios from 'axios'
+import { IamService } from './iam.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'angular-dapp';
+
+  constructor(
+    private readonly iamService: IamService
+  ) { }
+
+  isLoading: boolean = false
+  did = undefined
+  roles: Role[]
+
+  async verifyIdentity() {
+    const claim = await this.iamService.iam.createIdentityProof();
+    const {
+      data: { token }
+    } = await axios.post<{ token: string }>("https://did-auth-demo.energyweb.org/login", {
```

Comment:
It's okay for now but I would recommend to use most commonly used HTTPClient from @angular/common/http

Reply:
Thanks for the feedback, I will merge for now, and then fix at later point